simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -1045,29 +1248,30 @@ class simd<std::int32_t, simd_abi::avx2_fixed_size<4>> {
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION simd& operator=(simd const&) = default;
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION simd& operator=(simd&&) = default;
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION static constexpr std::size_t size() {
-    return 4;
+    return 8;
   }
-  template <class U, std::enable_if_t<std::is_convertible_v<U, value_type>,
-                                      bool> = false>
+  template <typename U, std::enable_if_t<std::is_convertible_v<U, value_type>,
```

Comment:
```suggestion
  template <class U, std::enable_if_t<std::is_convertible_v<U, value_type>,
```
Looks like we are almost exclusively using `class U` so far.

Reply:
Changed to `class U`.